src/core/server/elasticsearch/elasticsearch_service.ts

```diff
@@ -186,6 +187,7 @@ export class ElasticsearchService
       adminClient: this.adminClient,
       dataClient,
       createClient: this.createClient,
+      status$: calculateStatus$(esNodesCompatibility$),
```

Comment:
now we can stop exporting  `esNodesCompatibility$` ? https://github.com/elastic/kibana/blob/960d8a3c305f52307b298746d660777a58e71f00/src/core/server/saved_objects/saved_objects_service.ts#L400-L405

Reply:
It's still being used by the legacy ES plugin. I was going to switch that over in a separate PR.